### ISSUE TITLE
Not able to update Issue with issueLinks under 

### ISSUE THREAD

**Issue by pragmaticivan** [ctreminiom/go-atlassian#185]
**go-atlassian version**
Latest

**go-atlassian component**
- [x] Jira Software Cloud
- [ ] Jira Agile Cloud
- [ ] Jira Service Management Cloud 
- [ ] Confluence Cloud
- [ ] Atlassian Admin Cloud

**Describe the bug** :bug:
```go
	var payload = models.IssueSchemeV2{
		Fields: &models.IssueFieldsSchemeV2{
IssueLinks: []*models.IssueLinkScheme{
				{
					ID: issueLinkType,
					OutwardIssue: &models.LinkedIssueScheme{
						Key: "AA-1234",
					},
				},
			},
...

resp, err := jiraClient.Issue.Update(ctx, targetKey, false, &payload, &customFields, nil)
```

Causes `"{\"errorMessages\":[],\"errors\":{\"issuelinks\":\"Field does not support update 'issuelinks'\"}}"`

**Comment by ctreminiom** [comment]
Hi @pragmaticivan, 

Unfortunately, the issuelinks cannot be edited using the Edit Issue method due it's blocked by the Jira API. I would recommend you to use the the [Issue Links](https://docs.go-atlassian.io/jira-software-cloud/issues/link) methods.

Instead, the Jira API only supports the creation and deletion of issue links, not the edition.

We can add a new issue link using the Edit endpoint with the operation verb, but it's not supported by the library, yet.

**Reference**: https://confluence.atlassian.com/jirakb/how-to-use-rest-api-to-add-issue-links-in-jira-issues-939932271.html

**Comment by pragmaticivan** [comment]
Thx! I ended up performing an upsert with that api.
